### DISCUSSION TITLE
How do i disable the table of contents (toc) in a markdown file?

### DISCUSSION THREAD

**Discussion by harimohanr** [mkdocs/mkdocs#3235] [Q&A]
I am new to mkdocs. I dont want the table of contents in the left hand side of my index.md page. I am using the default mkdocs them. I have tried adding various metadata to my index.md to remove it. but its not working.

Things i have tried. None worked. Please help
---
toc: false
hide: 
       - navigation
       - toc
 no_toc: true
---

**Comment by redtide** [comment]
See also #2486

**Reply by kwameopareasiedu** [reply]
Here's how I hacked it. Let's say I wanted to hide TOC in my `index.md`.

1. Add some non-standard HTML tags to the file you'd like to hide markdown from. Since these tags will not be rendered, I used them as markers. In this example, I used `<hide_toc>`:

    `index.md`
    ```markdown
    # Hello World

    Here's my awesome site
    <hide_toc>
    ```

2. Create a custom javascript file with the following code:
    `js/hide_toc.js`
    ```javascript
    const hasHideTocTag = !!document.querySelector("hide_toc");

    if (hasHideTocTag) {
      const containerRow = document.querySelector("body > .container > .row");
      const tocCol = containerRow.querySelector("& > div:first-child");
      tocCol.style.visibility = "collapse";
    }
    ```

3. Register the custom javascript file in your `mkdocs.yml`
    `mkdocs.yml`
    ```yaml
    site_name: Site
    extra_javascript:
        - js/hide_toc.js
    ```

Now for every other page you want to hide TOC, just add `<hide_toc>` anywhere in the file. Hope this helps
